mlfinlab/sampling/bootstrapping.py

```diff
@@ -6,34 +6,40 @@
 from numba import jit, prange
 
 
-def get_ind_matrix(triple_barrier_events):
+def get_ind_matrix(triple_barrier_events, price_bars):
     """
     Snippet 4.3, page 64, Build an Indicator Matrix
```

Comment:
Add here in the docstring that we changed it to include the price_bars and why we did that.

Reply:
This will answer questions if readers want to know why the change from the book.